src/mcp_gateway/__main__.py

```
@@ -9,7 +16,9 @@ def main() -> None:
     try:
         run_gateway()
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)